x/participationrewards/keeper/msg_server_test.go

```diff
@@ -316,6 +316,36 @@ func (suite *KeeperTestSuite) Test_msgServer_SubmitClaim() {
 			&types.MsgSubmitClaimResponse{},
 			"",
 		},
+		{
+			"valid_osmosis_bank_proof",
```

Comment:
does bank proof mean we are looking at the account balance of userr submitting claim? seems misleading?